src/standalone/Soil/boundary_conditions.jl

```diff
@@ -439,89 +454,94 @@ end
 # Methods
 
 """
-    boundary_flux(bc::HeatFluxBC,  _...)::ClimaCore.Fields.Field
+    boundary_flux!(bc_field, bc::HeatFluxBC,  _...)
 
 A method of boundary fluxes which returns the desired flux.
 
 We add a field of zeros in order to convert the bc (float) into
 a field.
 """
-function boundary_flux(
+function boundary_flux!(
+    bc_field,
     bc::HeatFluxBC,
     boundary::ClimaLand.AbstractBoundary,
     model,
     Δz::ClimaCore.Fields.Field,
     Y::ClimaCore.Fields.FieldVector,
     p::NamedTuple,
     t,
-)::ClimaCore.Fields.Field
+)
     FT = eltype(Δz)
-    return FT.(bc.bc(p, t)) .+ FT.(ClimaCore.Fields.zeros(axes(Δz)))
+    bc_val = FT.(bc.bc(p, t))
+    @. bc_field = bc_field * 0
```

Comment:
Same as above

Reply:
Actually, why is this needed? You are writing a different value in the line below?

Reply:
youre right! I think we can just directly do:
`bc_field .= bc_val`

Reply:
thanks!